### ISSUE TITLE
NameError: name 'c2' is not defined

### ISSUE THREAD

**Issue by benlandrum** [sigma-py/quadpy#473]
Hello. I tried importing `quadpy` version `0.17.1`, but I ran into an import error. Could you please advise?

```
Python 3.10.10 (main, Feb  8 2023, 00:00:00) [GCC 12.2.1 20221121 (Red Hat 12.2.1-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import quadpy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.10/site-packages/quadpy/__init__.py", line 1, in <module>
    from . import c1
  File "/usr/local/lib/python3.10/site-packages/quadpy/c1/__init__.py", line 1, in <module>
    from .._agg import _hLoBZ as integrate_adaptive, _A3fFc as chebyshev_gauss_1, _dHchr as chebyshev_gauss_2, _LvUIZ as clenshaw_curtis, _hIGSI as fejer_1, _WTR0v as fejer_2, _Flu2i as gauss_jacobi, _w9Nwi as gauss_kronrod, _e0kVY as gauss_legendre, _QqYq0 as gauss_lobatto, _8IFO6 as gauss_patterson, _LhwRE as gauss_radau, _OJzqD as midpoint, _EFGLA as newton_cotes_closed, _U0r6B as newton_cotes_open, _m0lXp as trapezoidal
  File "/usr/local/lib/python3.10/site-packages/quadpy/_agg/__init__.py", line 2, in <module>
    x21.dex_23c(__file__)
  File "<string>", line 8302, in <module>
  File "<string>", line 4, in wrapper
  File "<string>", line 8209, in __load__
NameError: name 'c2' is not defined
```

**Comment by nschloe** [comment]
I'll look at it first thing tomorrow.

**Comment by nschloe** [comment]
Fixed now in 0.17.7. Digging out the bug revealed some other things, also fixed. One benefit is that (on the average) integration is now a little more precise for _all_ schemes.

Let me know if this fix works for you.

**Comment by benlandrum** [comment]
> Fixed now in 0.17.7. Digging out the bug revealed some other things, also fixed. One benefit is that (on the average) integration is now a little more precise for _all_ schemes.
> 
> Let me know if this fix works for you.

Thank you, Nico. It works.
